app/src/main/java/com/simon/harmonichackernews/StoryRecyclerViewAdapter.java

```diff
@@ -508,11 +504,6 @@ public class SubmissionsHeaderViewHolder extends RecyclerView.ViewHolder {
         public SubmissionsHeaderViewHolder(View view) {
```

Comment:
Usually headers are placed in Toolbars that [collapse on scroll](https://github.com/material-components/material-components-android/blob/master/docs/components/TopAppBar.md#applying-scrolling-behavior-to-the-top-app-bar). Might be worth refactoring later